front/app/containers/Admin/projects/project/participationContext/utils/votingMethodUtils.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactNode } from 'react';
+import BudgetingInputs from '../components/voting/votingMethodInputs/BudgetingInputs';
+import { VotingInputsProps } from '../components/VotingInputs';
+
```

Comment:
Created this new VotingMethodConfig which works the same way as the existing ParticipationMethodConfig does.